src/modules/navigator/rtl.cpp

```diff
@@ -246,9 +246,9 @@ RTL::set_rtl_item()
 			/* disable previous setpoint to prevent drift */
 			pos_sp_triplet->previous.valid = false;
 
-			mavlink_log_info(_navigator->get_mavlink_log_pub(), "RTL: descend to %d m (%d m above home)",
-					 (int)(_mission_item.altitude),
-					 (int)(_mission_item.altitude - _navigator->get_home_position()->alt));
+			mavlink_and_console_log_info(_navigator->get_mavlink_log_pub(), "RTL: descend to %d m (%d m above home)",
```

Comment:
Yes I think so, but right now RTL might be confusing to a new user.
We probably need another mechanism to show what the vehicle is doing outside of a planned mission. Then in QGC we can show the loiter circle or point we're flying towards.